### DISCUSSION TITLE
小白求助import css格式失效

### DISCUSSION THREAD

**Discussion by FranciF** [ant-design/ant-design-mobile#4188] [Q&A] [answered]
在import 'antd-mobile/dist/antd-mobile.css' 时报错Module not found: Can't resolve 'antd-mobile/dist/antd-mobile.css' 
package.json内容如下：

> {
  "name": "lease",
  "version": "0.1.0",
  "private": true,
  "dependencies": {
    "@testing-library/jest-dom": "^5.11.4",
    "@testing-library/react": "^11.1.0",
    "@testing-library/user-event": "^12.1.10",
    "antd-mobile": "^5.0.0-beta.17",
    "react": "^17.0.2",
    "react-dom": "^17.0.2",
    "react-router-dom": "^5.3.0",
    "react-scripts": "4.0.3",
    "web-vitals": "^1.0.1"
  },
  "scripts": {
    "start": "react-scripts start",
    "build": "react-scripts build",
    "test": "react-scripts test",
    "eject": "react-scripts eject"
  },
  "eslintConfig": {
    "extends": [
      "react-app",
      "react-app/jest"
    ]
  },
  "browserslist": {
    "production": [
      ">0.2%",
      "not dead",
      "not op_mini all"
    ],
    "development": [
      "last 1 chrome version",
      "last 1 firefox version",
      "last 1 safari version"
    ]
  }
}

**Comment by jacob-lcs** [comment, marked as answer]
看打包产物 antd-mobile 5.0.0-beta.17 已经没有单独的 css 文件了，直接引入 js 文件即可

按需加载官方文档：https://mobile.ant.design/zh/guide/import-on-demand

**Reply by FranciF** [reply]
感谢！我看的东西有点久了估计
